src/fsharp/TcGlobals.fs

```diff
@@ -1190,7 +1192,6 @@ type public TcGlobals(compilingFslib: bool, ilg:ILGlobals, fslibCcu: CcuThunk, d
 
   // We use 'findSysAttrib' here because lookup on attribute is done by name comparison, and can proceed
   // even if the type is not found in a system assembly.
-  member val attrib_IsByRefLikeAttribute  = findSysAttrib "System.Runtime.CompilerServices.IsByRefLikeAttribute"
```

Comment:
I guess it does make sense to not have the instance of the attribute here as it's only for IL types; we wanted to expand the checks to cover IL and F# versions of the IsByRefLikeAttribute.